index.bs

```diff
@@ -313,10 +313,7 @@ directory rather than a file or files. When specified, the behavior on
 the selection of a directory is as if all files with that directory as
 an ancestor were selected. In addition, the
 {{File/webkitRelativePath}} property of each {{File}} is set to a
-[=relative path=] from the shortest common ancestor directory of all
-selected files to that file, including that ancestor in the path.
-(Therefore this may be a subdirectory of the selected directory
-rather than the directory itself.)
+[=relative path=] from the selected directory to the file.
```

Comment:
Edge, Firefox, and Safari set a relative path from the parent of the selected directory.
So, webkitRelativePath includes the basename of the selected directory.

In the following example, the result would be:

- "to_upload/a/b/c/1.txt"
- "to_upload/a/b/c/2.txt"
- "to_upload/a/b/3.txt"

Reply:
Awesome, thanks - will update.

https://github.com/web-platform-tests/wpt/blob/master/entries-api/file-webkitRelativePath-manual.html will need updating. (I should see if there's a way to make the core of those manual tests re-usable as blink layout tests)

Reply:
See latest PS.